dataset/src/test/scala/frameless/functions/NonAggregateFunctionsTests.scala

```diff
@@ -1,10 +1,9 @@
 package frameless
 package functions
 import frameless.functions.nonAggregate._
-import org.apache.spark.sql.{ Column, Encoder }
+import org.apache.spark.sql.{Encoder, functions => untyped}
```

Comment:
It's renamed this way `{functions => sparkFunctions}` in two other places, this should probably be consistent (I know you just moved it around, but since we are at it...)